piker/ui/_event.py

```diff
@@ -0,0 +1,87 @@
+# piker: trading gear for hackers
```

Comment:
this is our event proxying from Qt subsystem.
basically, afaict this avoids 90% of the api *layers* used in `qtrio` but pretty much getting 90% of it's functionality -> getting Qt events relayed to async code over a mem chan.

Reply:
done.